lib/counter_culture/counter.rb

```diff
@@ -14,6 +14,7 @@ def initialize(model, relation, options)
       @foreign_key_values = options[:foreign_key_values]
       @touch = options.fetch(:touch, false)
       @delta_magnitude = options[:delta_magnitude] || 1
+      @execute_after_commit = options.fetch(:execute_after_commit, false)
```

Comment:
@mrclmrvn I agree, although if you look at @magnusvk original request in #120, he had wanted it to default to be in the transaction. IMO this would be a breaking change and require a major version bump, but as a non-maintainer, that falls out of the scope of a PR per the README.

Happy to default to `true` though if that's what the maintainer(s) want.